cmd/list/machinepool/cmd.go

```diff
@@ -153,6 +155,22 @@ func run(_ *cobra.Command, _ []string) {
 	writer.Flush()
 }
 
+func printAutoscaling(autoscaling *cmv1.MachinePoolAutoscaling) string {
+	if autoscaling != nil {
```

Comment:
Can this be an explicit `&false` or will it always be `nil` for a non-autoscaling machinepool?

Reply:
it will always be nil